MATLAB.gitlab-ci.yml

```diff
@@ -24,7 +25,7 @@
 # [2] https://www.mathworks.com/help/cloudcenter/ug/create-a-custom-matlab-container.html
```

Comment:
Line 17: Delete this # symbol and the empty line?  I don't think it's needed because there's already an empty line with no # symbol below.